src/App.tsx

```diff
@@ -21,91 +22,83 @@ import AddNewRoom from './features/Admin/Rooms/AddNewRoom/AddNewRoom';
-
-  const routes = createBrowserRouter([
-    {
-      path: "/",
-      element: <AuthLayout />,
-      errorElement: <NotFound />,
-      children: [
-        { index: true, element: <Login saveUserData={saveUserData} /> },
-        { path: "login", element: <Login saveUserData={saveUserData} /> },
-        { path: "register", element: <Register /> },
-        { path: "forget-password", element: <ForgetPass /> },
-        { path: "reset-password", element: <ResetPass /> },
-        // { path: "verify-user", element: <VerifyUser /> },
-        { path: "notfound", element: <NotFound /> },
-        { path: "change-password", element: <ChangePass /> },
-      ],
-    },
-    {
-      path: "/admin/home",
-
-      element: (
-        <ProtectedRoute userData={userData}>
-          <MasterLayout userData={userData} />
-        </ProtectedRoute>
-      ),
-      errorElement: <NotFound />,
-      children: [
-        { index: true, element: <Home /> },
-        { path: "users", element: <Users /> },
-        { path: "rooms", element: <Rooms /> },
-        { path: "rooms/add-room", element: <AddNewRoom /> },
-        { path: "ads", element: <Ads /> },
-
-        { path: "ads/add-ad", element: <AddNewAd /> },
-
-        { path: "facilities", element: <Facilities /> },
-
-        { path: "bookings", element: <Bookings /> },
-      ],
-    },
-    {
-      path: "/user/home",
-
-      element: (
-
-          <UserMasterLayout/>
-
-      ),
-      errorElement: <NotFound />,
-      children: [
-        { index: true, element: <UserHome /> },
-
-        { path: "explore", element: <ExplorePage /> },
-        { path: "fav", element: <FavouritesPage /> },
-        { path: "room-details/:roomId", element: <RoomDetails /> },
-        // { path: "users", element: <Users /> },
-
-        { path: "profile", element: <Profile /> },
-        // { path: "room-details/:roomId", element: <RoomDetails /> },
-      ],
-    },
-  ]);
+  const { userData , userRole, saveUserData}: IAuth = useContext(AuthContext);
+console.log(userRole);
 
   return (
-    <>
-
-      <RouterProvider router={routes} />
-    </>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/">
+          {/* Route par défaut pour UserHome */}
+          <Route path="/" element={<UserHome/> }/>
+          <Route path="login" element={<Login />} />
+          <Route path="register" element={<Register />} />
+          <Route path="forget-password" element={<ForgetPass />} />
+          <Route path="reset-password" element={<ResetPass />} />
+          <Route path="change-password" element={<ChangePass />} />
+          <Route path="notfound" element={<NotFound />} />
+        </Route>
+
+        {/* Routes pour l'administration */}
+        <Route
+          path="/admin/home"
+          element={
+            <ProtectedRoute userData={userData}>
+           <MasterLayout userData={userData} />
+         </ProtectedRoute>
+            // userData && userRole === "admin" ? (
```

Comment:
Don't forget to remove unused comments, and if any comment is need, you should make it in a common understandable language to other team members.